test/common/http/codec_impl_corpus/clusterfuzz-testcase-minimized-codec_impl_fuzz_test-5689430000795648

```diff
@@ -0,0 +1,62 @@
+actions {
```

Comment:
What fixes are needed to make this corpus entry run successfully?  I assume it's the changes to the fuzz harness?

Reply:
It does run successfully now, with the changes in the diff that close that reset stream when the response ends before the request does.